pkgs/by-name/bu/bustle/package.nix

```
@@ -71,6 +69,6 @@ stdenv.mkDerivation (finalAttrs: {
       aleksana
     ];
```

Can we please add the gnome-circle team as maintainers?

```suggestion
    ] ++ lib.teams.gnome-circle.members;
```